doc/developers.md

```diff
@@ -82,7 +82,29 @@ spdlog::set_level(spdlog::level::debug);
 spdlog::debug("Send help");
 ```
 
-## Where's the main ?
+### Compiling on UCL's Myriad cluster
+<details>
+
+  > **Warning**
+  > These instructions are a bit experimental. Please use with care (and report anything that's wrong here)!
+
+
+  
+  If you want to test changes on UCL's [Myriad](https://www.rc.ucl.ac.uk/docs/Clusters/Myriad/) (and/or don't have MATLAB on your pesonal machine) you can try these instructions
+  ```{.sh}
```

Comment:
Sorry for the drip drip changes.

I had another thought, should we also mention ssh key forwarding?

```suggestion
  
  If you want to test changes on UCL's [Myriad](https://www.rc.ucl.ac.uk/docs/Clusters/Myriad/) (and/or don't have MATLAB on your pesonal machine) you can try these instructions.
  Firstly, you will probably want to [forward you ssh agent](https://stackoverflow.com/questions/12257968/) for your github ssh key.
  To do this, you first need to run the following your **_local_** machine:
  ```{.sh}
  ssh-add -L # check your ssh agent is running
  ssh-add /path/to/your/github/key/id_rsa
  ssh -o ForwardAgent=yes your_user@myriad.rc.ucl.ac.uk
  ```

  And once you're on Myriad:

  ```{.sh}
  git clone git@github.com:UCL/TDMS.git
```

Reply:
Or is that excessive?